scripts/tls/kube-conf

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+
+function usage {
+    echo "USAGE: $0 DEST MASTER_IP"
+    echo "example: $0 dest/path 192.168.1.21"
+}
+
+if [ -z "$1" ] || [ -z "$2" ]; then
+    usage
+    exit 1
+fi
+
+DEST="$1"
+MASTER_IP="$2"
+ADMIN_CERT_BASE64=$(cat $DEST/admin.pem | base64 -w 0)
+ADMIN_KEY_BASE64="$(cat $DEST/admin-key.pem | base64 -w 0)"
+CA_CERT_BASE64="$(cat $DEST/ca.pem | base64 -w 0)"
+
+if [ -f "$DEST/kubeconfig" ];then
+  echo "$DEST/kubeconfig already exists"
+  exit 1
+fi
+
+cat << EOF > $DEST/kubeconfig
+apiVersion: v1
+kind: Config
+users:
+- name: coreos-baremetal
+  user:
+    client-certificate-data: ${ADMIN_CERT_BASE64}
+    client-key-data: ${ADMIN_KEY_BASE64}
+clusters:
+- name: coreos-baremetal
+  cluster:
+    certificate-authority-data: ${CA_CERT_BASE64}
+    server: https://${MASTER_IP}:443
+contexts:
+- context:
+    cluster: coreos-baremetal
+    user: coreos-baremetal
+  name: coreos-baremetal
+current-context: coreos-baremetal
```

Comment:
nit: having all items named the same could cause confusion. The general [upstream approach](http://kubernetes.io/docs/user-guide/kubeconfig-file/) is to suffix with `-context` `-user` `-cluster`